administrator/components/com_contact/Service/HTML/AdministratorService.php

```
@@ -104,7 +104,7 @@ public function association($contactid)
 	 *
 	 * @since   1.6
 	 */
-	public function featured($value = 0, $i, $canChange = true)
+	public function featured($value = 0, $i = 0, $canChange = true)
```

Given this functionally changes the code (this is no longer compulsory) I want to throw and InvalidArgumentException in this code if `$i===0` because in com_contact that is an impossible scenario and shouldn't be allowed

---

it also completely breaks the component